test/unittests/wasm_engine_test.cpp

```diff
@@ -230,3 +230,32 @@ TEST(wasm_engine, memory)
         EXPECT_FALSE(engine->init_memory(bytes(PageSize + 4, 0)));
     }
 }
+
+TEST(wasm_engine, host_function)
+{
+    /* wat2wasm
+    (func $extfunc (import "env" "crc32") (param i32) (result i32))
+    (memory (export "memory") 1)
+    (func $test (export "test") (param $a i32) (result i32)
+      local.get $a
+      call $extfunc
+    )
+    */
+    const auto wasm = from_hex(
+        "0061736d0100000001060160017f017f020d0103656e760563726333320000030201000503010001071102066d"
+        "656d6f72790200047465737400010a08010600200010000b");
+
+    for (auto engine_create_fn : all_engines)
+    {
+        auto engine = engine_create_fn();
+        ASSERT_TRUE(engine->parse(wasm));
+        ASSERT_TRUE(engine->instantiate(wasm));
```

Comment:
Ideally this test case would add the imported function here, but right now I would refrain from that because that introduces a call overhead in our abstraction. Maybe I'm wrong here, @gumb0 @chfast?

Reply:
I'm not sure this will help in the case were we have only single host function.

Reply:
For this PR it would not make any difference, however if the interface would be flexible, we could add more functions in fizzy-bench or have external tools use it, such as ewasm-bench, with a multitude of functions.